rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(book, person, _date)
+    @book = book
+    @person = person
+    book.rentals << self
+    person.rentals << self
+  end
```

Comment:
- [ ] Let's ensure to initialize `@date` instance in the `Rental` constructor so as to set its initial state when a new Rental is created, like below: 

```ruby
class Rental
  attr_accessor :date
  
  def initialize(date)
    @date = date
  end
end
```

Let's endeavor to read further on the same [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars)